app/views/pensions/incomeFromPensions/RemovePensionSchemeView.scala.html

```diff
@@ -0,0 +1,54 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.templates.Layout
+@import views.html.templates.helpers.Heading
+@import views.html.templates.helpers.Button
+@import views.html.templates.helpers.ErrorSummary
+@import views.html.templates.helpers.Link
+@import models.pension.statebenefits.UkPensionIncomeViewModel
+
+@this(
+        layout: Layout,
+        heading: Heading,
+        link: Link,
+        formWithCsrf: FormWithCSRF,
+        button: Button,
+        errorSummary: ErrorSummary,
+        govUkButton: GovukButton
+)
+
+@(taxYear: Int, pensionSchemeName: String, pensionSchemeIndex: Option[Int])(implicit request: AuthorisationRequest[_], messages: Messages, appConfig: AppConfig)
+
+@titleText = @{messages("incomeFromPensions.removePensionScheme.title", pensionSchemeName)}
+
+@layout(pageTitle = titleText, taxYear = Some(taxYear), isAgent = request.user.isAgent) {
+
+
+ @formWithCsrf(action = controllers.pensions.incomeFromPensions.routes.RemovePensionSchemeController.submit(taxYear, pensionSchemeIndex)) {
+  @heading(titleText, Some(messages("common.incomeFromPensions.caption", (taxYear - 1).toString, taxYear.toString)), "govuk-!-margin-bottom-2")
+
+  <div class="govuk-button-group">
+   @govUkButton(Button(
+    attributes = Map("data-module" -> "govuk-button", "id" -> "remove-employer-button-id"),
+    preventDoubleClick = true,
+    content = Text(messages("incomeFromPensions.removePensionScheme.buttonText")),
+    classes = "govuk-button--warning"
+   ))
+   @link(controllers.pensions.incomeFromPensions.routes.UkPensionIncomeSummaryController.show(taxYear).url, messages("common.cancel"), id = Some("cancel-link-id"))
+  </div>
+ }
+}
```

Comment:
theres some coverage missing for the view but think its because its missing this at the bottom like the other pages:
```
@{
// $COVERAGE-OFF$
} 
```